<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Regist</class>
 <widget class="QDialog" name="Regist">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>688</width>
    <height>384</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="user_name">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>200</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="user_password_0">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>100</y>
     <width>200</width>
     <height>20</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="user_password_1">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>130</y>
     <width>200</width>
     <height>20</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="user_phone">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>180</y>
     <width>200</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="user_email">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>210</y>
     <width>200</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="user_self_yu">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>270</y>
     <width>200</width>
     <height>20</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <item>
    <property name="text">
     <string>我的名字</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>我的生日</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>我的高中名字</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>我的大学在那</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>我的母亲名字</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>我父亲名字</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>我母亲生日</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>我父亲生日</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="user_self_in">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>270</y>
     <width>180</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="user_self_password">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>300</y>
     <width>200</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="user_save">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>300</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>保存注册信息</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="user_cancle">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>300</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>取消注册</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="user_background">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>30</y>
     <width>371</width>
     <height>261</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>688</width>
     <height>384</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>75</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;*用户名首字母不能为数字&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>user_name</zorder>
  <zorder>user_password_0</zorder>
  <zorder>user_password_1</zorder>
  <zorder>user_phone</zorder>
  <zorder>user_email</zorder>
  <zorder>user_self_yu</zorder>
  <zorder>user_self_in</zorder>
  <zorder>user_self_password</zorder>
  <zorder>user_save</zorder>
  <zorder>user_cancle</zorder>
  <zorder>user_background</zorder>
  <zorder>label_2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
